docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+    This is an open window within the meeting to discuss about topics
+    (even off-topics) not covered during a standard meeting.
+
+-   Agile Sprint meeting
+
+    Twice a month, in addition to the topics covered above, a Planning
+    Sprint and a Closing Sprint activities is inserted within the meeting.
+    All node administrators are asked to create a “work-item” for every
+    single activity they are going to carry out so that to have a trace of
+    the whole FIWARE Lab activities and effort spent.
+
+Beside all points above, a specific mailing list
+(fiware-lab-federation-nodes@lists.fiware.org) is constantly used from
+Node Administrators and FIWARE Lab experts to exchange each other
+doubts, information, tips and any kind of communication useful to the
+growth and stability of FIWARE Lab.
+
+### Community Account Requests
+
+A Community User is allowed to experiment with FIWARE technology for a
+period of more than 9 months. Typical examples are SMEs/start-ups under
+the FIWARE Accelerator Programme.
+
+Trial Users can always apply to for upgrading their accounts to become
+Community Users. This is granted to everybody if it is understood that
+the application they aim to developing, is considered a relevant
+reference example for the development of the FIWARE Community.
+
+In order to apply to become a Community User it is necessary to compile
+an application form accessible through the main page of the FIWARE Lab
+portal (<https://account.lab.fiware.org>) - click the *Request Community
+Account Upgrade* - button.
+
+![FIWARE Lab request community account upgrade.](image19.png)
+
+This will open a window in which information about application is
+requested from the user, this information will be used to understand
+exactly what is the planned use of the resources, why they are required
+and in which FIWARE Lab node you plan to use those resources.
+
+![FIWARE Lab community account request](image20.png)
+
+It will create a ticket inside the FIWARE Lab Upgrade Account in Jira to
+be response by the L1 Support team and assigned accordingly to the
+corresponding FIWARE Lab administrator node in order to resolve it.
+
+![FIWARE Lab Upgrade Account (FLUA) Jira project.](./media/image21.png)
+
+From here, the FIWARE Lab node administrators will take those tickets
+and apply the corresponding activities described in section 7.1.3 to
+provide the corresponding resources to the user.
+
+### Help Desk Support
+
+The Help Desk activities are daily part of FIWARE Lab operations. The
+Help Desk is the support that FIWARE Lab experts, Nodes Administrators
+and GEs Owners, give to external and internal users. The Help Desk
+activities are structured in 2 main Level of support in order to
+guarantee that agreed SLAs are achieved.
+
+The first level of support is comprised of a team which is in charge of
+managing all FIWARE Lab incoming tickets. This team is also responsible
+for categorizing all incoming tickets in order to guarantee that Backlog
```

Comment:
for categorizing all incoming tickets in order to guarantee that proper BA process can be executed to extract relevant information about the use of the different FIWARE GEs.

The rest of the sentence can be deleted...